lib/web_ui/dev/screenshot_manager.dart

```diff
@@ -0,0 +1,148 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// @dart = 2.6
+import 'dart:io' as io;
+import 'dart:convert';
+import 'dart:math';
+
+import 'package:image/image.dart';
+import 'package:webkit_inspection_protocol/webkit_inspection_protocol.dart'
+    as wip;
+import 'package:yaml/yaml.dart';
+
+import 'common.dart';
+
+/// [DriverManager] implementation for Chrome.
+///
+/// This manager can be used for both macOS and Linux.
+class ChromeScreenshotManager extends ScreenshotManager {
+  String get filenameSuffix => '';
+
+  Future<Image> capture(Map<String, dynamic> region) async {
+    final wip.ChromeConnection chromeConnection =
+        wip.ChromeConnection('localhost', kDevtoolsPort);
+    final wip.ChromeTab chromeTab = await chromeConnection.getTab(
+        (wip.ChromeTab chromeTab) => chromeTab.url.contains('localhost'));
+    final wip.WipConnection wipConnection = await chromeTab.connect();
+
+    Map<String, dynamic> captureScreenshotParameters = null;
+    if (region != null) {
+      captureScreenshotParameters = <String, dynamic>{
+        'format': 'png',
+        'clip': <String, dynamic>{
+          'x': region['x'],
+          'y': region['y'],
+          'width': region['width'],
+          'height': region['height'],
+          'scale':
+              1, // This is NOT the DPI of the page, instead it's the "zoom level".
```

Comment:
nit: this code is copy paste from the existing code. (Personal opinion: I believe the best way of handling this problem ui having a standard within the team rather than only asking some developers to fix the code. May be we can add one to engine repo so that everyone follows the same conventions).

Reply:
done.